HeterogeneousCore/SonicTriton/plugins/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<library   name="HeterogeneousCoreSonicTritonPlugins_converters" file="converters/*.cc">
+  <use   name="FWCore/Utilities"/>
+  <use   name="FWCore/ParameterSet"/>
+  <use   name="FWCore/PluginManager"/>
+  <use   name="DataFormats/Common"/>
```

Comment:
this include should go in the package BuildFile (rather than the plugin BuildFile) since it's used in the .h file